src/lib/components/Dialog.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+  let dialog: HTMLDialogElement;
+
+  export function close() {
+    dialog.close();
+  }
+
+  export function open() {
+    dialog.showModal();
+  }
```

Comment:
We should consider adding a listener for clicks outside the dialog to close the modal.